definitions/artifacts/azure-communication-service.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$md": {
+    "access": "public",
+    "name": "azure-communication-service"
+  },
+  "type": "object",
+  "title": "Azure Communication Service",
+  "description": "",
+  "additionalProperties": false,
+  "required": [
+    "data",
+    "specs"
+  ],
+  "properties": {
+    "data": {
+      "title": "Artifact Data",
+      "type": "object",
+      "required": [
+        "infrastructure",
+        "authentication",
+        "security"
+      ],
+      "properties": {
+        "infrastructure": {
+          "title": "Infrastructure configuration",
+          "type": "object",
+          "required": [
+            "ari"
+          ],
+          "properties": {
+            "ari": {
+              "$ref": "../types/azure-resource-id.json"
+            }
+          }
+        },
+        "authentication": {
+          "title": "Authentication credentials",
```

Comment:
What does an 'endpoint` look like? I don't see it as an attribute on the tf resource

Reply:
It looks like `https://local-test-dev-0001.communication.azure.com`

I'm realizing I should make a type for this instead of using https url type. I'll add that in tonight.

Reply:
Where do you get the value from in TF?

Reply:
Oh, I don't. I have to create it: `hostname = https://${azurerm_communication_service.main.name}.communication.azure.com`

Reply:
dont over optimize for a type we aren't using in multiple places.. its just a URL, if we want to enforce format their is a format type in json schema.

https://json-schema.org/understanding-json-schema/reference/string.html#uri-template

Lets call the field whatever azure calls it. its endpoint in the PR, but you just referred to it as hostname in the snippet above, but its technically a "URL" (not a hostname) in that example.

Reply:
> dont over optimize for a type we aren't using in multiple places.. its just a URL, if we want to enforce format their is a format type in json schema.

sounds good, ill swap it back to the `https-url.json` type

> Lets call the field whatever azure calls it. its endpoint in the PR, but you just referred to it as hostname in the snippet above, but its technically a "URL" (not a hostname) in that example.

ah that was just copy/pasta from the `artifacts.tf` file when it still used the API artifact definition. i will definitely use `endpoint` with the actual artifact

Reply:
for context
![image](https://user-images.githubusercontent.com/8037512/214698759-b296a7b4-c963-48e2-baee-848e94c10432.png)

Reply:
for context
![image](https://user-images.githubusercontent.com/8037512/214698915-9b3f1d47-2d82-4489-96e6-bc4ddd738dc6.png)